Land_Sector_Data_Analysis/Land_Sector_Dataset_Description.md

```diff
@@ -0,0 +1,290 @@
+*   The general notation of files is : **COUNTRY_AL4_STATE-NAME.json**( implying the country name followed by Administrative Level 4, follwed by the statename).
+The files are made up of eight (08) fields(	**country,	ISO3166_2,	name,	SNAME,	mg_code,	IPCC,	adminlevel,	geometry**)
+
+
+*  The notation of files with the general form: **COUNTRY_AL4_STATE-NAME_GEZ.json**(implying country name followed by administrative level 4, followed by state name; with GEZ: Global Economic Zone). The files have 08 fields(**country,	ISO3166_2,	name,	gez_name,	gez_code,	gez_abbrev,	adminlevel	geometry**)
+
+
+
+*   The notation of files with the general form: **COUNTRY_AL4_STATE-NAME_WSR.json**(implying country name followed by administrative level 4, followed by state name; with WSR: World Soil Resources). The files have 08 fields(	**country,	ISO3166_2,	name,	SNAME,	mg_code,	IPCC,	adminlevel,	geometry**)
+
+
+
+
+*   THe general notation of files is : **COUNTRY_AL4_STATE-NAME.json**( implying the country name followed by Administrative Level 4, follwed by the statename).
+The files are made up of eight (08) important fields: as explained below:
+> 1. **country:** refers to the name of the country
+> 2. **ISO3166_2:** (ISO standard codes of administrative divisions and subdivisions) In other words - Administrative Divisions of Countries
+> 3. **name:** name of the level 4 state
+> 4. **SNAME:** soil name standard code
+> 5. **mg_code:** the soil major group code
+> 6. **IPCC:** soil names(The Intergovernmental Panel on Climate Change)
+> 7. **adnminlevel:** administrative level
+> 8. **geometry**
+
+
+
+
+
+*  The notation of files with the general form: **COUNTRY_AL4_STATE-NAME_GEZ.json**(implying country name followed by administrative level 4, followed by state name; with GEZ: Global Economic Zone). The files have 08 fields. Some of which are:
+> 1. **gez_name:** Global Economic Zone name
+> 2. **gez_code:** Global Economic Zone standard code
+> 3. **gez_abbrev:** Global Economic Zone standard abbreviation
+
+
+
+*   The notation of files with the general form: **COUNTRY_AL4_STATE-NAME_WSR.json**(implying country name followed by administrative level 4, followed by state name; with WSR: World Soil Resources). The files have 08 fields(	**country,	ISO3166_2,	name,	SNAME,	mg_code,	IPCC,	adminlevel,	geometry**)
+
+
+
+
+> We also have two files that contain the world's country boundaries and the world's states boundaries respectively.
+
+
+
+
+### 1.2.   Protected Areas
+>This directory contains the boundary dataset for **World Database of Protected Areas WDPA** ezpecially data on Other Effective Area-Based Conservation Measures (WDOECM). A detailed description of the files within can be found[ here](http://wdpa.s3.amazonaws.com/WDPA_Manual/English/WDPA_Manual_1_4_EN_FINAL.pdf).
```

Comment:
```suggestion
This directory contains the boundary dataset for **World Database of Protected Areas WDPA.** Especially data on other effective area-based protection measures (WDOECM). A detailed description of the files in it can be found [here.](http://wdpa.s3.amazonaws.com/WDPA_Manual/English/WDPA_Manual_1_4_EN_FINAL.pdf).
```